packages/renderers-js/public/templates/fragments/instructionParseFunction.njk

```diff
@@ -33,7 +33,7 @@ export function {{ instructionParseFunction }}<
       & IInstructionWithAccounts<TAccountMetas>
     {% endif %}
     {% if hasData %}
-      & IInstructionWithData<Uint8Array>
+      & IInstructionWithData<ReadonlyUint8Array>
```

Comment:
Nice! I think this is the solution to the type error here, right? https://codesandbox.io/p/sandbox/dfsktf